src/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import CommentInfo from '../CommentInfo/CommentInfo';
+import { Comment } from '../types';
+
+type Props = {
+  comments: Comment[];
+
+};
+
+const CommentList:React.FC<Props> = ({ comments }) => {
+  return (
+    <div>
+      COMMENTS
+      {comments ? comments.map(comment => <div key={comment.id}><CommentInfo comment={comment} /></div>) : ''}
```

Comment:
```suggestion
      {comments && comments.map(comment => (
           <CommentInfo  key={comment.id} comment={comment} />
       )}
```

Reply:
no need for the additional wrapper, add key to component
also format it right
and use logical operators in this case